packages/osd-babel-preset/webpack_preset.js

```diff
@@ -36,6 +36,18 @@ module.exports = () => {
         {
           useBuiltIns: 'entry',
           modules: false,
+          targets: [
+            'Firefox > 0 and last 2 years and > 0.01%',
+            'Chrome > 0 and last 2 years and > 0.01%',
+            'Safari > 0 and last 2 years and > 0.01%',
+            'Edge > 0 and last 1 years and > 0.01%',
+            'Opera > 0 and last 2 years and > 0.01%',
+            '> 0.2%',
+            'not op_mini all',
+            'not and_uc < 100',
+            'not android < 100',
+            'not dead',
+          ],
```

Comment:
Would it make sense to dynamically load the browserlist here?

Reply:
What a great idea! lemme do it.